src/test/java/dk/bec/bookanything/UserControllerTest.java

```diff
@@ -0,0 +1,25 @@
+package dk.bec.bookanything;
+
+
+import dk.bec.bookanything.controller.UserController;
+import dk.bec.bookanything.service.UserService;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.web.servlet.WebMvcTest;
+import org.springframework.boot.test.mock.mockito.MockBean;
+import org.springframework.test.context.junit.jupiter.SpringExtension;
+import org.springframework.test.web.servlet.MockMvc;
+
+@ExtendWith(SpringExtension.class)
+@WebMvcTest(UserController.class)
+public class UserControllerTest {
```

Comment:
So many tests missing, please add unit tests for at least every controller and every service